### DISCUSSION TITLE
CSV Plugin: Null values plotted as 0

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#66183] [Plugins]
If I am using the CSV plugin with a file that looks like:
```
datetime, indoor_w_solar, indoor_no_solar, indoor_lux, outdoor_w_solar, outdoor_no_solar, outdoor_lux
2023-04-05 20:10, 4.11, 4.11, 99, , ,  
2023-04-05 22:36, 4.09, 4.09, 0, , ,  
2023-04-06 07:23, 4.04, 4.04, 3300, , ,  
2023-04-06 18:11, 4.00, 3.99, 700, , ,  
2023-04-07 08:21, 3.93, 3.92, 18500, , ,  
2023-04-07 10:15, 3.93, 3.91, 34000, , ,  
2023-04-07 12:10, 3.93,	3.90, 3000, 4.06, 4.09, 62000
```
The null values on the first several rows all got plotted as 0 in a Time Series panel or show as 0 in a Table. Why? They should be not plotted at all, just as they would not be if the data were pulled from a database.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
